app/views/thredded/messageboards/_form.html.erb

```diff
@@ -29,3 +29,8 @@
       </li>
     </ul>
 <% end %>
+
+<%= button_to "Delete", messageboard_path(messageboard), :method => 'delete',
+:class => "thredded--messageboard--delete", :form_class => "thredded--form",
```

Comment:
Style / formatting:

1. Use single quotes (`'`).
2. Avoid hash-rocket (`=>`), e.g. use `class:` instead of `=>`.
3. Indent arguments so they start on the same column (as `"Delete"`)

Reply:
Done.